WORKSPACE

```diff
@@ -21,78 +21,143 @@ workspace(name = "vaticle_typedb_driver")
 # Load @vaticle_dependencies #
 ##############################
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_bazel_rules_go",
+    integrity = "sha256-fHbWI2so/2laoozzX5XeMXqUcv0fsUrHl8m/aE8Js3w=",
+    urls = [
+        "https://mirror.bazel.BUILD/github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+    ],
+)
+
+http_archive(
+    name = "bazel_gazelle",
+    integrity = "sha256-MpOL2hbmcABjA1R5Bj2dJMYO2o15/Uc5Vj9Q0zHLMgk=",
+    urls = [
+        "https://mirror.bazel.BUILD/github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
+
+load("//go:deps.bzl", "go_dependencies")
+# gazelle:repository_macro go/deps.bzl%go_dependencies
+go_dependencies()
+go_rules_dependencies()
+go_register_toolchains(version = "1.21.10")
```

Comment:
It's a bad practice to hardcode values as parameters (just like in programming). Let's put this version as a variable somewhere and get this variable from `WORKSPACE`. 
Check out
`load("//python:python_versions.bzl", "register_all_toolchains")` (where to put) and `load("@vaticle_dependencies//builder/antlr:deps.bzl", antlr_deps = "deps", "antlr_version")` (how to put and load)

Reply:
Let's refactor `dotnet_register_toolchains` for practice as well, it doesn't look good now =)